src/containers/Login/index.js

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable react/prefer-stateless-function */
+import React, { Component } from 'react'
+import {
+  Button, Col, Form, Input, Layout, Row, Typography, Alert
+} from 'antd'
+import { UserOutlined, LockOutlined } from '@ant-design/icons'
+import { connect } from 'react-redux'
+import { Redirect } from 'react-router-dom'
+import { LOGIN_SUBMIT } from '../../store/auth/actions/types'
+
+class Login extends Component {
+
+  loginSummitHandler (values) {
+    const { username, password } = values;
+    this.props.login(username, password);
+  }
```

Comment:
if you plan to use `component/container` pattern you can start to use it here
also be careful with formatting )

Reply:
done